general-superstaq/general_superstaq/user_config.py

```diff
@@ -153,7 +155,8 @@ def _config_to_yaml_str(config: Any) -> str:
         return self._client.aqt_upload_configs({"pulses": pulses_yaml, "variables": variables_yaml})
 
     def aqt_get_configs(self) -> Dict[str, str]:
-        """Calls private client to get the configurations for AQT
+        """Calls private client to get the configurations for AQT.
+
         Returns:
             The configurations as a Dictionary
```

Comment:
nit: let's make this a bit more descriptive, because this isn't obvious from the source. e.g.
```suggestion
            A dictionary containing all of the user's configs (as YAML strings), indexed by the config names (e.g. "pulses", "variables").
```